crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -402,6 +402,15 @@ impl Account {
     /// > subsequently claiming to have sent messages which they didn't.
     /// > sender must correspond to the user who sent the event, recipient to
     /// > the local user, and recipient_keys to the local Ed25519 key.
+    ///
+    /// # Arguments
+    ///
+    /// * `sender` -  The `sender` field from the top level of the received
+    ///   event.
+    /// * `sender_key` - The `sender_key` from the cleartext `content` of the
```

Comment:
One thing to note perhaps, even though this is coming from the cleartext, it is guaranteed to be correct because the decryption was successful.

Reply:
added some words to this effect.